c++/cpp2py/converters/vector.hpp

```diff
@@ -46,14 +45,14 @@ namespace cpp2py {
 
   template <typename T> struct py_converter<std::vector<T>> {
 
-    static PyObject *c2py(std::vector<T> const &v) {
+    static PyObject *c2py(std::vector<T> v) {
 
       if constexpr (has_npy_type<T>) {
-        return make_numpy_proxy_from_vector(v).to_python();
+        return make_numpy_proxy_from_vector(std::move(v)).to_python();
       } else { // Convert to Python List
         PyObject *list = PyList_New(0);
         for (auto const &x : v) {
-          pyref y = py_converter<T>::c2py(x);
+          pyref y = py_converter<std::decay_t<T>>::c2py(std::move(x));
```

Comment:
Confused. 
Where is the other change to make an array of obj ? Another PR ?

Reply:
Yes, this will be another PR